source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,89 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it need to be done on an account by account basis. **NOTE** Macie ONLY looks at S3 buckets and checks their access information so it will not look at databases or EC2 settings, for example.
+
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one. In theory you can just set it up in each account and allow one (say production) to use every account to access their data. At the time this document was produced this was not tested but can be used according to an AWS video showing a [demo](https://www.youtube.com/watch?v=8piwEQJJXdo).
```

Comment:
This is to do with manual set up isn't it so can be removed?

Reply:
It is manual set up. I could mention that in the documentation if you would prefer. If not I will remove it